src/wp-includes/script-loader.php

```diff
@@ -1312,6 +1312,8 @@ function wp_default_scripts( $scripts ) {
 
 	$scripts->add( 'wp-api', "/wp-includes/js/wp-api$suffix.js", array( 'jquery', 'backbone', 'underscore', 'wp-api-request' ), false, 1 );
 
+	$scripts->add( 'webp-shim', "/wp-includes/js/webp-fallback$suffix.js", array(), '1.0.0', 1 );
```

Comment:
This script should probably not be registered, since it should never be enqueued in the regular way. It should behave similar to WP core's `wp-emoji-release.js` script, which is only being loaded from another JS inline script tag. That script is not regsitered in WP's script loader either.

Reply:
Make sense.